sdf.md

```diff
@@ -1116,6 +1139,15 @@ groupings made mandatory within groupings containing them.
 The affordance/grouping itself that carries the sdfRequired keyword is declared
 to be mandatory to be represented.
 
+Note that text strings with a bare affordance or grouping name are not
+subject to JSON pointer encoding as discussed in {{hierarchical-names}}.
+To ensure that they are reliably distinguished from JSON pointers,
+these text strings cannot contain ":" or "#" characters.
+(If these characters are contained in such a name, a JSON pointer
+needs to be formed instead, potentially requiring further path
+elements as well as JSON pointer encoding, the need for which is best
+avoided by choosing a Given Name without these characters.)
```

Comment:
This is an important design consideration that is explained, but it reads a bit like : or # could be used, but shouldn't rather than that they are forbidden.

Reply:
I rephrased this by using the term "referenceable-name" from the grammar.